L1Trigger/VertexFinder/python/TPStubValueMapProducer_cff.py

```diff
@@ -1,30 +1,30 @@
 import FWCore.ParameterSet.Config as cms
-from L1Trigger.VertexFinder.VertexNTupler_cff import L1TVertexNTupler
-from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
+from L1Trigger.VertexFinder.VertexNTupler_cff import l1tVertexNTupler
```

Comment:
`VertexNTupler_cff` was renamed to `VertexNTupler_cfi`, so it looks like this import will fail.

Reply:
Fixed